src/style.js

```diff
@@ -13,7 +13,7 @@ const style = css`
     --mmp-media-cover-info-color: var(--mini-media-player-media-cover-info-color, --mmp-text-color);
     --mmp-text-color-inverted: var(--disabled-text-color);
     --mmp-active-color: var(--mmp-accent-color);
-    --mmp-icon-color: var(--mini-media-player-base-color, var(--paper-item-icon-color, #44739e));
+    --mmp-icon-color: var(--mini-media-player-icon-color, var(--paper-item-icon-color, #44739e));
```

Comment:
I would be ok with this change if you make the initial fallback `mini-media-player-base-color` and then  `paper-item-icon-color` etc...